src/devices/machine/z80sio.cpp

```diff
@@ -1,5 +1,5 @@
 // license:BSD-3-Clause
-// copyright-holders:Curt Coder, Joakim Larsson Edstrom
+// copyright-holders:Curt Coder, Joakim Larsson Edstrom, F.Ulivi
```

Comment:
I’m not sure this warrants claiming copyright, considering bigger reworkings (e.g. 8c9602f24091274c60a829a8fc99df268d6c3565) haven’t resulted in claiming copyright.